app/models.py

```diff
@@ -158,7 +160,7 @@ def password(self, password):
     def check_password(self, password):
         return check_hash(password, self._password)
 
-    def get_permissions(self, service_id=None):
+    def get_permissions(self, service_id=None, organisation_id=None):
```

Comment:
Why is the organisation_id included here? It doesn't appear to be used.

Reply:
Good spot - looks like a leftover from something I tried initially. Will remove :)